pkg/standalone/uninstall.go

```diff
@@ -6,13 +6,29 @@ import (
 	"github.com/dapr/cli/utils"
 )
 
-func Uninstall() error {
+func Uninstall(uninstallAll bool) error {
 	err := utils.RunCmdAndWait(
 		"docker", "rm",
 		"--force",
 		DaprPlacementContainerName)
+
+	errorMessage := ""
 	if err != nil {
-		return errors.New("Dapr Placement Container may not exist")
+		errorMessage += "Could not delete Dapr Placement Container - it may not have been running "
+	}
+
+	if uninstallAll {
+		err = utils.RunCmdAndWait(
```

Comment:
In uninstall -all - you could iterate the list of configured containers and remove and print/list the uninstalled container in the loop.

Reply:
I merged the PR before this comment, but I think it's fine to wait to update that later if there are others.